### PR TITLE
Todo-List

### PR DESCRIPTION
**Pull request by evabanegacom**
This is a JavaScript web app 
The concept here is a todo-list using document object model to write HTML and css
We made the interface
Added the logic and functionality
Included active storage

### CODE REVIEW

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required.
Great job guys :clap: 
The data has been made persistent across sessions using local storage.
Let's make some changes in order to meet all the requirements.

**REQUIRED**
*  DOM-Manipulation and Logic are in separate modules ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/modules_requirement.png) from Odin). Currently, logic related code has not been separated with DOM manipulative code.
* There is a 'default' project. Let's add a default list in the local storage.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/projects_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/todo-list)).
* Let's make sure that a todo item is an object which is dynamically generated using a factory or a constructor. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/todo_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/todo-list)).

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathgo.**

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi team, 👋 

### Great job thus far! 🙌 Kindly fix the requirement of the previous reviewer before submitting for another CRR. 🥺 

**KIndly submit another CRR once these issues have been solved.** _Happy coding!_ 🎉

**Comment by BrittanyBlake** [comment]
Thank you for the review.  
We have implemented the changes requested however, in our original code, we already had the todo item as an object. If you check lines 29-37 of our list.js file, you can see the object and if you check lines 5-13 of our todos.js, you can see the class constructor. We look forward to your comments.

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello Team
### You have done well so far :clap: However, you have a few issues to fix before the project can be approved.
- [ ] Add edit feature on the to do items
- [ ] Fix the linting errors raised on the pull request
- [ ] Separate app logic from dom manipulation
### Make these changes and submit for another code review. Good luck.
- Happy hacking!

By Raymond
Slack @iamraymond

File: src/js/list.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  };
+
+  function objectValues() {
+    const storageObject = localStorageList();
+    const storageObjectMap = Object.keys(storageObject).map((key) => {
+      const allTasks = storageObject[key];
+      return allTasks;
+    });
+    return storageObjectMap;
+  }
```
Comment (rOluochKe) [inline src/js/list.js]: ### separate app logic from dom manipulation

**Pull request closed without merging**